test/CMakeLists.txt

```diff
@@ -966,6 +974,39 @@ COMMAND ${DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --ver
 COMMAND ${DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --verbose --half --input  1 1 8 8 --weights 1 1 2 2 --pads_strides_dilations 0 0 1 1 2 2 --disable-forward --disable-backward-data
 COMMAND ${DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --verbose --half --input  1 128 56 56 --weights 1 128 5 5 --pads_strides_dilations 0 0 2 2 1 1 --disable-forward --disable-backward-data
 )
+add_custom_test(test_conv_igemm_dynamic_xdlops_nhwc_fwd SKIP_UNLESS_ALL HALF_ENABLED GFX908_ENABLED VEGA_DISABLED
+COMMAND ${DYNAMIC_IMPLICITGEMM_XDLOPS_NHWC_FWD_ENVS} $<TARGET_FILE:test_conv2d> --verbose --input  64 256  7  7 --weights 128 256 1 1 --pads_strides_dilations 0 0 1 1 1 1 --disable-backward-data --disable-backward-weights --in_layout NHWC --fil_layout NHWC --out_layout NHWC
```

Comment:
This won't test HALF. See #991

Reply:
Hi @atamazov I checked https://github.com/ROCmSoftwarePlatform/MIOpen/pull/991, basically I need to change to use `... $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} ...` in every line of cmd right? and this need to be done after https://github.com/ROCmSoftwarePlatform/MIOpen/pull/991 is merged

Reply:
It looks like so.

Reply:
OK, got it

Reply:
@atamazov  I merged the latest the develop and changed to `${MIOPEN_TEST_FLOAT_ARG}`, in latest commit. Please review again thanks